uui-core/src/data/processing/views/tree/LoadableTree.ts

```diff
@@ -249,6 +249,10 @@ export abstract class LoadableTree<TItem, TId> extends EditableTree<TItem, TId>
             if (newNodesCount !== nodeInfo.count) {
                 nodeInfo = { ...nodeInfo, count: newNodesCount };
             }
+
+            if (parent == null && response.totalCount != null && nodeInfo.totalCount !== response.totalCount) {
```

Comment:
nodeInfo.totalCount !== response.totalCount check looks unnecessary here

Reply:
Probably we don't need this 'if' at all - I don't see why should we ignore totalCount on non-root nodes.

One case I see, is that API might want to query server for totalCount once per list, and don't re-query it for each page/each (topIndex, visibleCount). However, if we allow this, e.g. by assuming totalCount=undefined to mean "keep previous value", I'm afraid of some stale-cache issues...

Reply:
@jakobz, it makes sense. Done.